supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -228,6 +228,22 @@ When combined with privileges, permissions define a user's or resource's overall
 
 *See also*:
 
+[[playbook]]
+==== image:images/yes.png[yes] playbook (noun)
+*Description*: Playbooks are Ansible's configuration, deployment, and orchestration language.
+Playbooks can describe a policy you want your remote systems to enforce, or a set of steps in a general IT process.
```

Comment:
```suggestion
Playbooks can describe a policy you want your remote systems to enforce or a set of steps in a general IT process.
```
- Remove comma after subordinating conjunction `or`. A comma would only be necessary if  an independent clause followed `or`.

Reply:
Another copy over from "Ansible Playbook". I'll update that entry also.